nautobot/dcim/tests/test_api.py

```diff
@@ -688,9 +688,10 @@ def setUpTestData(cls):
             status=statuses[0],
         )
         # Place a device in Rack 4
-        device = Device.objects.filter(
-            location=populated_rack.location, rack__isnull=True, device_type__u_height__gt=0
-        ).first()
+        device = Device.objects.filter(location=populated_rack.location, rack__isnull=True).first()
+        # Ensure the device height is 1U
```

Comment:
What problem is this trying to prevent? Does it have to be 1 or does it just need to be non-zero? If that's the case we should say that in this comment.

Reply:
Just non-zero, but in all the factories, it only set it to 0, 1 or 2.  I thought I would just pick one.

Reply:
Just non-zero, but in all the factories, it only set it to 1 or 2.  I thought I would just pick one.